packages/devtools/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## 0.0.15 - Date TBD
-* Warn users when they should be using a profile build of their application instead of a debug build.
```

Comment:
why did this get removed from the changelog?
also add an entry for the browser detect cl.

Reply:
Ah my mistake. I was misreading an item included in the last release and thought they were the same. Re-added and included browser detection. 

Browser detection was included in the last release, but I made this note Microsoft specific.

"Warn users using Microsoft browsers (IE and Edge) that they should be using Chrome to run DevTools."